ext/index.d.ts

```diff
@@ -0,0 +1,5 @@
+import * as priority from './priority'
```

Comment:
`priority` should be marked as deprecated.

Reply:
I think this is outside the scope of typing `ext`. I also don't know what your mechanism for deprecation is. I only found a line comment saying deprecated.

Reply:
I meant using the `@deprecated` annotation as described in https://palantir.github.io/tslint/rules/deprecation/. Some legacy features should no longer be used as they have better replacements and documenting both could cause confusion about which construct to use. By explicitly declaring them as deprecated, it will be clear they should no longer be used.

Since there should be an explanation for each deprecated feature, it may difficult for you to add without knowledge of the history of these deprecations, so I agree it's out of scope of the PR.

Reply:
Did you want to keep this PR open until the deprecations are added? I think the typings could already be useful as they are.

Reply:
Yes, sorry for the radio silence. Let me review again as I'd like to include it in the next release.